web/modules/custom/dpl_event/src/PriceFormatter.php

```diff
@@ -67,22 +99,32 @@ public function formatPriceRange(array $prices): string {
     $filtered_prices = array_filter($prices, fn($price) => $price > 0);
 
     // Determine the highest price in the range.
-    $highest_price = max($filtered_prices);
-
-    // Format the highest price.
-    $formatted_highest_price = $this->formatRawPrice((string) $highest_price);
+    $highest_price_raw = max($filtered_prices);
 
     // Construct the price range string.
     if ($has_free_price) {
+      // Get the price, with prefix and suffix, as we only use this one.
+      $highest_price_formatted = $this->formatPrice((string) $highest_price_raw);
+
       // Only display "Free" and the highest price.
-      return $this->translation->translate("Free") . ' - ' . $formatted_highest_price . ' kr.';
+      return $this->translation->translate("Free") . " - $highest_price_formatted";
```

Comment:
Again a matter of taste, but I thin `sprintf` is easier to read than using the `.`string concatenation:
```php
      return sprintf("%s - %s", $this->t("Free"), $highest_price_formatted);

Reply:
I tried, but PHPStorm informs me that i need to introduce a whole package, which i think is overkill :)

```
phpstan: Function sprintf is unsafe to use. It can return FALSE instead of throwing an exception. 
Please add 'use function Safe\sprintf;' at the beginning of the file to use the variant provided by the 'thecodingmachine/safe' library.

Reply:
You just need to do like this:
https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/b5565c74fd2dfe3426c69e37641bdea73b38dfd2/web/modules/custom/dpl_url_proxy/src/Form/ProxyUrlConfigurationForm.php#L10